classes/Configuration/Configuration.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace PrestaShop\Module\PrestashopCheckout\Configuration;
+
+use Module;
+
+class Configuration
+{
+    /**
+     * @var Module
+     */
+    private $module;
+
+    /**
+     * @param Module $module
+     */
+    public function __construct(Module $module)
+    {
+        $this->module = $module;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isAdmin()
+    {
+        return defined('_PS_ADMIN_DIR_');
+    }
+
+    /**
+     * @return bool
+     */
+    public function isDebug()
+    {
+        return (bool) constant('_PS_MODE_DEV_');
+    }
+
+    /**
+     * @return int
+     */
+    public function getMaxLoggerFiles()
+    {
+        return 15;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPrestaShopPath()
+    {
+        return (string) constant('_PS_ROOT_DIR_');
```

Comment:
I guess you use the method `constant` to prevent PHP warnings, but when we look at php.net, it seems the methods triggers a warning anyway : https://www.php.net/manual/en/function.constant.php

Reply:
Fixed 👍